README.md

```diff
@@ -1,6 +1,19 @@
 # Stedi EDI Bootstrap
 
-This repo contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one possible way to interact with Stedi's Products and APIs to achieve a typical EDI workload; your implementation may include some or all of these products depending on your particular systems and requirements.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
+
+
+# Read and Write Workflow 
+
+The bootstrap uses the following process to handle incoming EDI:
+
+1. An example trading partner uses [Stedi SFTP](https://www.stedi.com/products/sftp) to send EDI documents to a [Stedi Bucket](https://www.stedi.com/products/buckets).
+2. New documents automatically invoke a [Stedi Function](https://www.stedi.com/products/functions) that contains custom code for the workflow. 
+3. The function calls [Stedi EDI Translate](https://www.stedi.com/products/edi-translate) to transform the EDI data into JSON. This process uses a [Stedi Guide](https://www.stedi.com/products/guides) to map EDI fields to JSON fields. 
```

Comment:
I don’t add _Stedi_ to product that start with EDI. I don’t think we decided on a naming scheme, but that’s what I use at the moment. So, Stedi Buckets, but EDI Translate. I started using the _Stedi_ prefix to avoid confusion (since our products have generic names like Buckets, Functions, SFTP, etc.), but I don’t think that’s necessary for names like EDI Translate and EDI Core; no risk of confusion there.

Reply:
Yeah this is something I keep going back and forth on, so I'm really glad you brought it up. My opinion is that we should always use Stedi in front of the ambiguous ones like you have already been doing, and then be able to use it front of other product names if it's the first reference to the product/if required in context. I realize this may be a point of contention haha, but just saying my thoughts on this. I'm going to look into the way other folks handle this because I want it to be in our editorial style guide. I might leave it in this copy unless you feel VERY strongly about me removing it.

Reply:
I think I can manage to suppress my outrage over your flaunting of my personal and arbitrary, but clearly superior, convention around naming our products. :wink:

Reply:
hahaha Joost you're awesome